cli/create/git.go

```diff
@@ -19,40 +19,32 @@ func initGitRepo(projectCtx *project.ProjectCtx) error {
 	// check that git is installed
 	if !common.GitIsInstalled() {
 		return fmt.Errorf("git not found. " +
-			"You'll need to add the app to version control yourself later")
+			"You'll need to add the application to version control yourself later")
 	}
 
-	// init repo
+	log.Debug("Initialize empty git repository")
 	initCmd := exec.Command("git", "init")
 	if err := common.RunCommand(initCmd, projectCtx.Path, false); err != nil {
-		return fmt.Errorf("Failed to initialize git repo")
+		return fmt.Errorf("Failed to initialize git repository")
 	}
 
-	log.Debug("Initialized git repo")
-
-	// add files to index
+	log.Debug("Add files to index")
```

Comment:
May be `Add files to git index`? Or add git prefix for such messages

Reply:
There is `Initialize application git repository` info level message above, I think it's enough. 

I've changed this message to `Add files to git index` anyway.